src/pages/createPost.tsx

```diff
@@ -1,35 +1,28 @@
-import React from 'react'
-import {useState} from 'react'
+import React from "react";
 import Navbar from "@/components/public/Navbar";
-import { editProfileHeader } from "public/locales/editProfileHeader";
+import { useState } from "react";
```

Comment:
you can import it in the same line as `React` (in line 1) as `import React, { useState } from "react";`